src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -88,6 +88,63 @@ export function DAppsExplorerScreen() {
     setHelpDialogVisible(false)
   }
 
+  // create sections for all screen content to facilitate scroll to top using a SectionList
+  const sections = []
```

Comment:
i had to refactor how we generate sections to enable the scroll to top functionality on tap of the toast. for a section list, you can only scroll to sections / section items but the top of this screen is at the top of the section list header. the only way is to specify an offset _in pixels_ to account for the header, which doesn't work for us if the favorites list is in the header since its length is dynamic.

https://reactnative.dev/docs/sectionlist#scrolltolocation

Reply:
I don't fully get this, could we scroll to the top of the whole list instead? The favorite should always be visible from the very top, right?
Sorry may be missing some details on the behavior.

Reply:
Ok thanks for the explanation.
`SectionList` is using a ScrollView under the hood,
what about using `sectionListRef.current?.getScrollResponder()?.scrollTo({ y: 0, animated: true })` instead?

Reply:
that works! i didn't see this in the docs but i see it in the types def now, thanks for the suggestion! i've updated :)